### ISSUE TITLE
Unresolvable maven-dependency com.hivemq:hivemq-extension-sdk:4.4.2-SNAPSHOT

### ISSUE THREAD

**Issue by fnordian** [hivemq/hivemq-community-edition#220]
## Expected behavior
Adding com.hivemq:hivemq-community-edition-embedded:2020.5 as dependency should not result in error.

## Actual behavior

Adding com.hivemq:hivemq-community-edition-embedded:2020.5 as dependency results in `Cannot resolve com.hivemq:hivemq-extension-sdk:4.4.2-SNAPSHOT`

### Reproducer code
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>org.example</groupId>
    <artifactId>embed-hivemq-test</artifactId>
    <version>1.0-SNAPSHOT</version>

    <build>
        <plugins>
            <plugin>
                <artifactId>maven-compiler-plugin</artifactId>
                <configuration>
                    <source>14</source>
                    <target>14</target>
                </configuration>
            </plugin>
        </plugins>
    </build>

    <dependencies>
        <dependency>
            <groupId>com.hivemq</groupId>
            <artifactId>hivemq-community-edition-embedded</artifactId>
            <version>2020.5</version>
        </dependency>
    </dependencies>
</project>
```

## Details
- Affected HiveMQ CE version(s): hivemq-community-edition-embedded 2020.5
- Used JVM version: 14

It seems that this can be worked around with excluding com.hivemq:hivemq-extension-sdk:4.4.2-SNAPSHOT and adding com.hivemq:hivemq-extension-sdk:4.4.2 as explicit dependency:
```
 <dependencies>
        <dependency>
            <groupId>com.hivemq</groupId>
            <artifactId>hivemq-community-edition-embedded</artifactId>
            <version>2020.5</version>
            <exclusions>
                <exclusion>
                    <groupId>com.hivemq</groupId>
                    <artifactId>hivemq-extension-sdk</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>com.hivemq</groupId>
            <artifactId>hivemq-extension-sdk</artifactId>
            <version>4.4.2</version>
        </dependency>
    </dependencies>
```

**Comment by SgtSilvio** [comment]
Hi @fnordian 
Thank you for opening this issue and providing detailed information.
The pom.xml for 2020.5 wrongfully contains 4.4.2-SNAPSHOT. We will take care for the next version.
In the meantime your workaround (excluding and adding the extension sdk yourself) is a viable solution.

**Comment by SgtSilvio** [comment]
@fnordian we just released 2020.6 which resolved this issue.
